scripts/azure-dev.ps1

```
@@ -235,38 +275,53 @@ function Write-AzureDevSshInstructions {
   if (@($existingKeys).Count -eq 0) {
-    return
+    return $LocalSshPublicKey
   }
 
-  if ($SshPublicKey -in @($existingKeys)) {
-    return
-  }
+  # Azure keeps the original osProfile key immutable. Additional workstation
+  # keys live in the guest authorized_keys file, so setup preserves the live
+  # infrastructure key instead of requiring every workstation to possess it.
+  return [string](@($existingKeys) | Select-Object -First 1)
```

PSScriptAnalyzer (1.25.0)

[info] The cmdlet 'Get-AzureDevDeploymentSshPublicKey' returns an object of type 'System.String' but this type is not declared in the OutputType attribute.

(PSUseOutputTypeCorrectly)